ui/src/modules/Circles/Comparation/Item/Layer/Release.tsx

```diff
@@ -40,13 +42,22 @@ const LayerRelease = ({ circle, onClickCreate }: Props) => {
     </Styled.Release>
   );
 
+  const checkIfButtonIsDisabled = () => {
+    if (!circle?.id) {
+      return true;
+    } else if (!releaseEnabled) {
+      return true;
+    }
+    return false;
```

Comment:
Make sense if change to this?
```suggestion
    return (!circle?.id || !releaseEnabled)
```

Reply:
no, because if the first condition is true (case of editing) the button will always be enabled.

Reply:
and we need to check if the Id exists as well as the percentage available.